howto/upgrade/upgrade-anbox.md

```diff
@@ -5,23 +5,28 @@
 If you're interested in getting notified for the latest Anbox Cloud releases, make sure you subscribe to the [Anbox Cloud category](https://discourse.ubuntu.com/c/anbox-cloud/49) on the Ubuntu discourse.
 ```
 
-Anbox Cloud allows upgrades from older versions to newer version. This describes the steps necessary to perform the upgrade.
+Anbox Cloud officially supports only the latest release. This means that we support upgrades from n-1 to nth minor version, where n is the most recent minor version released.
 
-The upgrade instructions detail the revisions each charm needs to be upgraded to, to bring it to the latest version. Next to the upgrade of the charms any used images or addons need to be updated as well.
+If you are on an earlier version of Anbox Cloud than the most recent one, we recommend upgrading to the latest version as soon as possible. If you are multiple versions behind the supported version, there could be potential disruptions if you upgrade directly to the current version. You should keep this in consideration while planning your upgrade.
```

Comment:
Maybe we could also mention that `if an existing deployment is several versions behind the most recent one, the best practice is to upgrade incrementally, version by version.`, what do you think?